starlette/applications.py

```diff
@@ -73,9 +82,7 @@ def __init__(
     def build_middleware_stack(self) -> ASGIApp:
         debug = self.debug
         error_handler = None
-        exception_handlers: typing.Dict[
-            typing.Any, typing.Callable[[Request, Exception], Response]
-        ] = {}
+        exception_handlers: _ExcDict = {}
```

Comment:
It's not possible to not use an alias here, as that would require the TypeVars to be bound to the class.